### PR TITLE
add mini styleguide for ui components

### PR DESCRIPTION
**Pull request by brookeangel**
To do: 
- [x] Add Muli font
- [x] ignore elm.js

### CODE REVIEW

**Comment by jwoudenberg** [comment]
Hey @brookeangel, I finally added those last two points we were missing here. Do those changes make sense to you? If so I feel we can merge this!

File: styleguide-app/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# NoRedInk style guide
```
Comment (brookeangel) [inline styleguide-app/README.md]: Yay, thanks for the readme!!! 😻 Could we add a link to this from the main Readme to let people know how to test their new styles?

**Comment by jwoudenberg** [comment]
Thanks Brooke! I'll follow up on your comments tomorrow and get this merged!

**Comment by jwoudenberg** [comment]
Texts look beautiful now!

screen shot 2018-02-21 at 15 50 20

**Pull request merged**